scss/_patterns_card.scss

```diff
@@ -16,14 +20,14 @@
     }
 
     &__footer {
-      border-top: 1px solid $color-mid-dark;
+      border-top: 1px solid $color-mid-light;
       padding-top: 1rem;
     }
   }
 
   .p-card {
     @extend %p-card;
-    border: 1px solid $color-mid-dark;
+    border: 1px solid $color-mid-light;
```

Comment:
This will need to be moved to the card placeholder as both `p-card` and `p-card--highlighted` require this border.